util/events/libuv/FileNo.c

```diff
@@ -93,9 +97,17 @@ static void incoming(uv_pipe_t* stream,
     } else {
         struct FileNo_pvt* fileno = Identity_check((struct FileNo_pvt*) stream->data);
         if (fileno->peer.accepted_fd != -1) {
+            int oldFd = fileno->peer.accepted_fd;
+            int newFd = 123;
```

Comment:
@lgierth I _think_ we used `dup2()` just so we know what fd value to look for during debug? It's working now with `dup()`.